.github/CODEOWNERS

```diff
@@ -42,7 +42,7 @@
 /go/ @zeroshade
 /java/ @lidavidm
 /js/ @domoritz @trxcllnt
-/matlab/ @assignUser
+/matlab/ @assignUser @kevingurney
```

Comment:
```suggestion
/matlab/ @kevingurney
```
As I only added myself to catch any stray matlab PRs that need you tagged on I can now be removed, congrats again!

Reply:
Thank you, @assignUser! :)